libethcore/Common.cpp

```diff
@@ -191,5 +189,26 @@ string TransactionSkeleton::userReadable(bool _toProxy, function<pair<bool, stri
 		formatBalance(gas * gasPrice) + ".");
 }
 
+LogEntry::LogEntry(RLP const& _r)
+{
```

Comment:
Should there be an assertion against
- `_r` not being a list and
- `_r` being too short?